src/arith/rewrite_simplify.cc

```diff
@@ -474,6 +474,8 @@ PrimExpr RewriteSimplifier::Impl::VisitExpr_(const DivNode* op) {
     if ((div(ramp(b1, c1, lanes), broadcast(c2, lanes))).Match(ret)) {
       int64_t c1val = c1.Eval()->value;
       int64_t c2val = c2.Eval()->value;
+      // If divisor is equal to zero
+      ICHECK(c2val != 0) << "division by zero";
```

Comment:
nit: The check itself is clear enough so we probably don't need the comment.

```suggestion
      ICHECK(c2val != 0) << "division by zero";
```